app/src/main/res/values-it/preferences.xml

```diff
@@ -1,18 +1,76 @@
-
+    <!--
+    Synchronize with @strings/preference_default_value_alarm_time_value
+    -->
+    <!--
+    Synchronize with @integers/preference_default_value_alarm_time_index
+    and string-array/preference_entry_values_alarm_time
+    -->
+    <string name="preference_dialog_title_alarm_time">Scegli un tempo predefinito per la sveglia</string>
+    <string name="preference_title_alarm_time">Scegli un tempo predefinito per la sveglia</string>
+    <string-array name="preference_entries_alarm_time_titles">
+        <item>@string/alarm_time_title_at_start_time</item>
+        <item>@string/alarm_time_title_5_minutes_before</item>
+        <item>@string/alarm_time_title_10_minutes_before</item>
+        <item>@string/alarm_time_title_15_minutes_before</item>
+        <item>@string/alarm_time_title_20_minutes_before</item>
+        <item>@string/alarm_time_title_30_minutes_before</item>
+        <item>@string/alarm_time_title_45_minutes_before</item>
+        <item>@string/alarm_time_title_60_minutes_before</item>
+    </string-array>
+    <!--
+    Synchronize with @strings/preference_default_value_alarm_time_value
+    and @integers/preference_default_value_alarm_time_index
+    -->
     <!-- Alarm tone -->
-
+    <string name="preference_title_alarm_tone">Seleziona il tono della sveglia</string>
+    <string name="preference_summary_alarm_tone">Apre la schermata di selezione dei suoni da qui.</string>
+    <!-- Device time zone -->
+    <string name="preference_title_use_device_time_zone_enabled">Usa fuso orario del dispositivo</string>
+    <string name="preference_summary_use_device_time_zone_enabled">Visualizza date e orari di conseguenza.</string>
     <!-- Alternative highlighting -->
-
+    <string name="preference_title_alternative_highlighting_enabled">Abilita evidenziazione alternativa</string>
+    <string name="preference_summary_alternative_highlighting_enabled">Aggiunge un bordo intorno alle sessioni preferite.</string>
     <!-- Alternative schedule URL -->
-
+    <string name="preference_title_alternative_schedule_url">Configura l\'URL alternativo del programma</string>
+    <string name="preference_summary_alternative_schedule_url">Inserire un URL di backup se il server preconfigurato non può essere raggiunto.</string>
+    <!-- App notification settings -->
+    <string name="preference_title_app_notification_settings">Personalizza le notifiche</string>
+    <string name="preference_summary_app_notification_settings">Open the associated system settings screen from here.</string>
     <!-- Insistent session alarms -->
-
+    <string name="preference_title_insistent_alarms_enabled">Abilita sveglia insistente</string>
+    <string name="preference_summary_insistent_alarms_enabled">Keeps an alarm on infinitely.</string>
+    <!-- Category Engelsystem -->
+    <string name="preference_summary_engelsystem_json_export_url"><![CDATA[Per vedere i tuoi turni inserisci qui il tuo <u>URL personale </u>.<br/><br/>
```

Comment:
Something seems to be wrong here:

```
/home/runner/work/EventFahrplan/EventFahrplan/app/src/main/res/values-it/preferences.xml:69:4: Failed to flatten XML for resource 'preference_summary_engelsystem_json_export_url' with error: Invalid unicode escape sequence in string
> Task :app:createCcc36c3DebugCompatibleScreenManifests
"{str}"
/home/runner/work/EventFahrplan/EventFahrplan/app/src/main/res/values-it/preferences.xml:69:4: string/preference_summary_engelsystem_json_export_url does not contain a valid string resource.
```

Reply:
@johnjohndoe should work now. Seems like it was a escaping problem with the italian l'